package.json

```diff
@@ -72,6 +72,7 @@
     "eslint-config-origami-component": "1.0.0",
     "eslint-config-prettier": "^2.9.0",
     "eslint-plugin-prettier": "^2.6.1",
+    "form-data": "^2.3.3",
```

Comment:
Should there be a change to package-lock.json with this?

Reply:
It's already included as a dependency of `request`